core/templates/dev/head/components/top_navigation_bar/TopNavigationBarDirective.js

```diff
@@ -100,9 +99,19 @@ oppia.directive('topNavigationBar', [function() {
           $scope.$apply();
           // Close the sidebar, if necessary.
           SidebarStatusService.closeSidebar();
+          $scope.sidebarIsShown = SidebarStatusService.isSidebarShown();
+        });
+        $scope.isSidebarShown = (function() {
+          if (SidebarStatusService.isSidebarShown()) {
+            angular.element(document.body).addClass('stop-scroll');
+          } else {
+            angular.element(document.body).removeClass('stop-scroll');
+          }
+          return SidebarStatusService.isSidebarShown();
+        });
+        $scope.toggleSidebar = (function() {
```

Comment:
I think we were talking about parenthesis which were outer to function, can you check.